### ISSUE TITLE
Accessibility- assets image view

### ISSUE THREAD

**Issue by ajanec01** [craftcms/cms#7386]
### Overview of accessibility issues in the image edit view

- When the edit view is activated the keyboard focus is not moved to the menu options; it’s also possible to interact with elements beneath the edit view
- No focus indication
- Rotate and crop options are not keyboard accessible; they also have the nrv issue- should be menuitems see a[ria menubar example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/menubar/menubar-2/menubar-2.html)
![menubar](https://user-images.githubusercontent.com/47136246/104527292-44d71d00-55fc-11eb-828a-329d316d4e68.png)
- Slider for image rotation is not keyboard accessible; it also has the nrv issue- [the vertical aria slider example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/slider/slider-2.html) seems to be a nice choice
![slider](https://user-images.githubusercontent.com/47136246/104527346-6cc68080-55fc-11eb-86bb-be5d85d5ea54.png)
- Focal point pointer is not keyboard accessible; also nrv issue; not sure how that feature could be made more accessible to sr users without overcomplicating things, a grid over the image?
- Landscape and portrait radio buttons are not labelled and not grouped; a more logical focus flow would be for them to appear before the cropping options as the options change in response to orientation
![radio-buttons](https://user-images.githubusercontent.com/47136246/104527414-92ec2080-55fc-11eb-9991-a3a8c1191596.png)
- Cropping options are not keyboard accessible; also nrv issue- could be listbox items as in the [aria listbox example ](https://www.w3.org/TR/wai-aria-practices-1.2/examples/listbox/listbox-scrollable.html)
- Custom crop option inputs are not labelled and not grouped
![custom-crop-inputs](https://user-images.githubusercontent.com/47136246/104527479-b44d0c80-55fc-11eb-9018-8822d45e9e8b.png)
- Status update for image size not conveyed
![status-update-text-image](https://user-images.githubusercontent.com/47136246/104527497-c333bf00-55fc-11eb-91a1-29953e87f2de.png)
- Image size value is an image of text- should be plain text so that it is possible to resize it; also, I think that this value should be constant for a given crop option; every time I use the text zoom in feature it changes- don't think that is correct

### Additional info

- Craft version: 3.5.17.1

**Comment by gcamacho079** [comment]
I’m closing this issue as most of the issues have been resolved as of version 4.4:
- When opening the Image Editor, the keyboard focus is moved into the modal and trapped inside until closed.
- Focus indicators use a light-on-dark color scheme to improve visibility.
- Rotate and crop now implement a vertical tab button pattern. When one of the buttons is focused, either up/down or left/right keypresses will activate the other tab panel.
- The slider for image rotation is now keyboard accessible and uses the APG slider pattern.
- Landscape and Portrait radio buttons are now grouped in a `fieldset`, and their inputs have accessible names. In addition, they now come before the cropping options in the DOM and visual order.
- Cropping options now use radio buttons and are keyboard accessible.
- Custom crop option inputs are now labeled.

Looking at the above list, I think only one issue hasn’t been resolved: the keyboard and screen reader accessibility of the focal point selection. However, accessibility fixes for that are on our roadmap, and I’ll update here when that is complete.
